src/views/collection-view/components/collection-tutorial-list/index.tsx

```diff
@@ -0,0 +1,36 @@
+import classNames from 'classnames'
+import TutorialCard from 'components/tutorial-card'
+import { TutorialCardPropsWithId } from 'components/tutorial-card/types'
+import s from './collection-tutorial-list.module.css'
+
+function CollectionTutorialList({
+  tutorials,
+  isOrdered,
+}: {
```

Comment:
Total style nit, but might be worth extracting into own interface.

Reply:
✅ Great call - done!